.npmrc

```diff
@@ -0,0 +1,3 @@
+registry=https://registry.npmjs.org/
+@ctt:registry=https://registry.npmjs.org/
+//registry.npmjs.org/:_authToken=f5069651-a619-4e87-8251-88042b707465
```

Comment:
Store npm credentials as hidden file in your home directory.

Reply:
done